tsconfig.json

```diff
@@ -6,4 +6,4 @@
     "downlevelIteration": true
   },
   "include": ["**/*.ts"],
```

Comment:
```suggestion
  "include": ["**/*.ts"],
  "exclude": [
    "./dist/**/*"
  ]
```

Reply:
I found out we need to explude the dist directory from the input of the tsc. Otherwise the build will error if there are already files in dist dir.